### PR TITLE
Refine UI.

### PR DESCRIPTION
**Pull request by AnyOfYou**

### CODE REVIEW

**Comment by madeye** [comment]
Thanks!

**Comment by Mygod** [comment]
This design has a usability problem. Please fix it.

Problem

**Comment by Mygod** [comment]
Also the floating action buttons lack proper ripples. Please fix those as well.

**Comment by AnyOfYou** [comment]
OK, I'm working on it.

**Comment by Mygod** [comment]
Great. Thanks. :smile:

File: src/main/res/drawable/ic_camera_alt.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FFFFFFFF"
+        android:pathData="M9 2L7.17 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2h-3.17L15 2H9zm3 15c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5z"/>
```
Comment (Mygod) [inline src/main/res/drawable/ic_camera_alt.xml]: You are right. `1.1.9` caused the crash here. Assigning this to myself.

**Comment by Mygod** [comment]
Btw where did you get those vector files? Those are extremely outdated - at least before Nov. 12, 2015.

**Comment by AnyOfYou** [comment]
I get those from [here](https://design.google.com/icons). I try to downloaded [this svg](https://design.google.com/icons/#ic_camera_alt) just now, still is the old version.

**Comment by Mygod** [comment]
Use xml files provided here please: https://github.com/google/material-design-icons

**Pull request merged**